### ISSUE TITLE
Projects Page

### ISSUE THREAD

**Issue by davideimola** [Schroedinger-Hat/schroedinger-hat-website#16]
We want to showcase our community projects inside a page on the site.

In the main page we will shown all the projects paged. We want to filter the projects by their categories.

We want a search bar where it can be possible to filter for name or tags.

When the user clicks on a project we want to show a page with the infos of the selected project.

Each project may have the following infos:

- Name (required)
- Description (required)
- Tags
- Category Type (event, sw, etc) (required)
- List of Project owners, with picture, name and surname (required)
- Social links of the projects
	- GitHub
	- Homepage
	- Discord channel
	- Reddit

It would be awesome if we can handle the projects as markdown files.

**Comment by TheJoin95** [comment]
We should install a markdown parser as dependecy to achieve that, but I agree on having a markdown file.
The main issue here is how to translate a markdown file via vue-i18n.

Perhaps we can achieve it by parsing the markdown filename choosing the file by the language code as a suffix or something similar.

Yep, the website is translated in english / italian and it's demanded to the navigator language

**Comment by davideimola** [comment]
Good point @TheJoin95! I have missed the i18n translation in my analysis.
